packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -0,0 +1,1085 @@
+            details.globalPosition - getOffset(context),
+            widget.transformationController.value,
+          ),
+        ));
+      },
+      onVerticalDragEnd: widget.onVerticalDragEnd,
+      onVerticalDragCancel: widget.onVerticalDragCancel,
+      onHorizontalDragDown: widget.onHorizontalDragDown == null ? null : (DragDownDetails details) {
+        widget.onHorizontalDragDown(DragDownDetails(
+          globalPosition: fromViewport(
+            details.globalPosition - getOffset(context),
+            widget.transformationController.value,
+          ),
+        ));
+      },
+      onHorizontalDragStart: widget.onHorizontalDragStart == null ? null : (DragStartDetails details) {
+        widget.onHorizontalDragStart(DragStartDetails(
+          globalPosition: fromViewport(
+            details.globalPosition - getOffset(context),
+            widget.transformationController.value,
+          ),
+        ));
+      },
+      onHorizontalDragUpdate: widget.onHorizontalDragUpdate == null ? null : (DragUpdateDetails details) {
+        widget.onHorizontalDragUpdate(DragUpdateDetails(
+          globalPosition: fromViewport(
+            details.globalPosition - getOffset(context),
+            widget.transformationController.value,
+          ),
+        ));
+      },
+      onHorizontalDragEnd: widget.onHorizontalDragEnd,
+      onHorizontalDragCancel: widget.onHorizontalDragCancel,
+      onPanDown: widget.onPanDown == null ? null : (DragDownDetails details) {
+        widget.onPanDown(DragDownDetails(
+          globalPosition: fromViewport(
+            details.globalPosition - getOffset(context),
+            widget.transformationController.value,
+          ),
+        ));
+      },
+      onPanStart: widget.onPanStart == null ? null : (DragStartDetails details) {
+        widget.onPanStart(DragStartDetails(
+          globalPosition: fromViewport(
+            details.globalPosition - getOffset(context),
+            widget.transformationController.value,
+          ),
+        ));
+      },
+      onPanUpdate: widget.onPanUpdate == null ? null : (DragUpdateDetails details) {
+        widget.onPanUpdate(DragUpdateDetails(
+          globalPosition: fromViewport(
+            details.globalPosition - getOffset(context),
+            widget.transformationController.value,
+          ),
+        ));
+      },
+      onPanEnd: widget.onPanEnd,
+      onPanCancel: widget.onPanCancel,
+      onScaleEnd: _onScaleEnd,
+      onScaleStart: _onScaleStart,
+      onScaleUpdate: _onScaleUpdate,
+
+      // Wrapping a Widget in an InteractiveViewer does not change how the
+      // widget is initially rendered. It should look identical whether or not
+      // the InteractiveViewer is there, until the transformation is changed.
+      child: Transform(
+        transform: widget.transformationController.value,
+        child: KeyedSubtree(
+          key: _childKey,
+          child: widget.child,
+        ),
+      ),
```

Comment:
Here's where I'm using this snippet instead (I don't need rotation):

````dart
      child: Stack(
        children: <Widget>[
          Positioned(
            left: widget.transformationController.value.row0.w,
            top: widget.transformationController.value.row1.w,
            child: KeyedSubtree(
              key: _childKey,
              child: Transform.scale(
                alignment: Alignment.topLeft,
                scale: widget.transformationController.value.row0.x,
                child: widget.child,
              ),
            ),
          ),
        ],
      ),
```

Reply:
Ah I see what you mean, thanks.  It seems like translation and rotation with the Transform widget cause drop targets to not work.  I'll try to look into this before I release this code.